app/src/renderer/components/common/AppTheme.vue

```diff
@@ -0,0 +1,57 @@
+<template lang="pug">
+#app-theme
+</template>
+
+<script>
+import { mapGetters } from "vuex"
+export default {
+  name: "app-theme",
+  computed: { ...mapGetters(["config"]) },
+  methods: {
+    setTheme(theme) {
+      if (theme === "light") {
+        this.setThemeLight()
+      } else {
+        this.setThemeDark()
+      }
+    },
+    setThemeLight() {
+      this.setCssVar("app-fg", "#eee")
```

Comment:
What do you think about getting these variables from some external files?

Reply:
I'm in favor of that. They're stored in AppTheme.vue right now just for convenience. The end goal is to let users import JSON theme files to change everything to their liking. Perhaps it can just be a simple JSON object with no nesting.

Reply:
@faboweb I can do it right now. Where should we store these files?

Reply:
`./app/src/renderer/vuex/json` perhaps?